frontend/src/locales/scripts/en.json5

```
@@ -1000,7 +1000,7 @@
     additional: "Coming soon",
     statusBeta: "Beta",
     seeImage: "See all images",
-    seeAudio: "See all audio",
+    seeAudio: "See all audio track",
```

Please use the plural form.
```suggestion
    seeAudio: "See all audio tracks",
```